<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmParametersSetting</class>
 <widget class="QWidget" name="frmParametersSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frameMain">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>400</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnNewConfig">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>204</width>
      <height>44</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEquipment">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>204</width>
      <height>44</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>204</width>
      <height>44</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="leftMenuWidget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>204</width>
      <height>325</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="btnLeftMenu2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>166</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLeftMenu3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>166</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLeftMenu4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>166</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLeftMenu1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>166</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnDevice">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>204</width>
      <height>44</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
